### PR TITLE
feat: android mpv support

### PR DESCRIPTION
**Pull request by Tigermouthbear**
This is a small change which allows the mpv-android player to be used without having to make any configuration changes like discussed in issue #164. I am willing to maintain Android support, so if anything breaks for Android in the future just let me know.

### CODE REVIEW

File: lobster.sh [hunk 1]
```diff
@@ -663,6 +664,8 @@ EOF
     if [ "$player" = "mpv" ] && ! command -v mpv >/dev/null; then
         if command -v mpv.exe >/dev/null; then
             player="mpv.exe"
+        elif echo "$(uname -a)" | grep -q "ndroid" 2>/dev/null; then
```
Comment (justchokingaround) [inline lobster.sh:667]: do this here:
https://github.com/justchokingaround/lobster/blob/main/lobster.sh#L47-L50

**Comment by justchokingaround** [comment]
thx for the PR, unfortunately i just merged v4.3.0 in which i rewrote a big part of the script (https://github.com/justchokingaround/lobster/commit/942ad9f466a1056fe48961826c1ff34da4beb976). could you please address my one comment and pull down the latest changes?

**Comment by justchokingaround** [comment]
great, i will test it later just to make sure its functional. can you bump the version number pls?

**Comment by Tigermouthbear** [comment]
so I got it updated to 4.3 and it works now, but I also had to change all instances of /tmp to use the TMPDIR environment variable because /tmp is not accessible on android

**Comment by justchokingaround** [comment]
are you sure this works fine in termux? it doesn't seem to properly work for me, but i did some tweaks to my mpv on android and to termux a while ago, so it could be smtg from my side. could u just test the latest commit and confirm it works pls?

**Comment by Tigermouthbear** [comment]
It is working for me, I also just updated the install instructions to place the script in the termux path. What error are you getting?

**Comment by justchokingaround** [comment]
the mpv intent starts but exits immediately. anw that is most likely a problem on my side. thanks for your contribution!

**Review by justchokingaround — APPROVED** [review]

**Review by justchokingaround — APPROVED** [review]

**Review by Tigermouthbear — COMMENTED** [review]
@justchokingaround the shellcheck commit broke the script for me on android. The elif statement wont execute without the echo.

**Comment by justchokingaround** [comment]
:sob: can u fix it, or should i do it?

i'm traveling for the next 2 days so i won't be able to do it until then

**Pull request merged**